### ISSUE TITLE
Fix CI failure

### ISSUE THREAD

**Issue by gimite** [google/personfinder#354]
CI started failing recently:
https://travis-ci.org/google/personfinder/builds/218793760

Looks like zlib is missing in the Docker image? Not sure why it suddenly started failing, though.

**Comment by RajibHossen** [comment]
I want to work on this issue. Could you please provide me some guidelines or right directions about it?

**Comment by AwesomeTurtle** [comment]
It seems that Pillow was updated and now zlib is required. So we need to update Dockerfile and install zlib.

**Comment by RajibHossen** [comment]
I am working on this.

**Comment by AwesomeTurtle** [comment]
This [issue](https://github.com/python-pillow/Pillow/issues/1412) can help. 

We have two solutions: 

+ Install zlib
+ Use a specific `Pillow` version

**Comment by RajibHossen** [comment]
Hello, My host machine is Ubuntu 16.04. In my host machine, I can successfully build the image from Dockerfile, then I run the image
`~/Sources/pf$ sudo docker build -t personfinder-local .`
`~/Sources/pf$ sudo docker images`
`[sudo] password for rajib: `
`REPOSITORY           TAG                 IMAGE ID            CREATED             SIZE`
`personfinder-local   latest              ef96376d6411        2 minutes ago       822 MB`
`phusion/baseimage    latest              b6b482650b50        2 weeks ago         247 MB`
`~/Sources/pf/docker$ ./run-container.sh personfinder-local`
`root@86fda3c30dac:/opt/personfinder# pwd`
`/opt/personfinder`
`root@86fda3c30dac:/opt/personfinder# ./docker/gae-run-app.sh` 
`Starting Person Finder server`
`Setting datastore_path to /tmp/8000.datastore`
`INFO     2017-04-05 12:37:31,346 devappserver2.py:764] Skipping SDK update check.`
`WARNING  2017-04-05 12:37:31,886 simple_search_stub.py:1146] Could not read search indexes from /tmp/appengine.googlepersonfinder.root/search_indexes`
`INFO     2017-04-05 12:37:31,887 api_server.py:268] Starting API server at: http://172.17.0.2:49532`
`INFO     2017-04-05 12:37:31,930 dispatcher.py:199] Starting module "default" running at: http://172.17.0.2:8000`
`INFO     2017-04-05 12:37:31,931 admin_server.py:116] Starting admin server at: http://172.17.0.2:8001`

I believe pillow error occur in windows or mac. If this is the case then how should I proceed to solve this issue?

**Comment by AwesomeTurtle** [comment]
All right, I have an idea : 

At each build, Travis downloads the latest version of `phusion/baseimage`. 

Your version is past from 2 weeks, so your version of `phusion/baseimage` seems working. We need to freeze a specific version of `phusion/baseimage` :

```
FROM phusion/baseimage
```

becomes

```
FROM phusion/baseimage:<WORKING VERSION>
```

**Comment by RajibHossen** [comment]
Hello, I changed the pillow version to 3.1.0 and docker build fails,then I again changed the version to 4.1.0
`RUN pip install --upgrade pip && pip install pytest lxml cssselect pillow==4.1.0`
and this built successfully.
Should I commit the changes and see if build works?

**Comment by AwesomeTurtle** [comment]
Add a Travis Integration on your PersonFinder fork and you will be able to run your build on your own CI. 😄 
If it works on your own Travis. You can submit a PR

**Comment by RajibHossen** [comment]
I think it's working with pillow 4.1.0
[PR](https://github.com/google/personfinder/pull/355)
Would you take a look. Also I will integrate Travis in my fork and will test.

**Comment by gimite** [comment]
Fixed with #355. Thanks!

**Comment by RajibHossen** [comment]
Integrated Travis in my own fork. It works as expected.
Thanks @AwesomeTurtle
